src/pages/api/boundary.ts

```diff
@@ -0,0 +1,41 @@
+import { Prisma } from '@prisma/client'
+import { NextApiRequest, NextApiResponse } from 'next'
+import { isProd } from 'src/app/_components/utils/isEnv'
+import { prismaClientForRawQueries } from 'src/prisma-client'
+import { z } from 'zod'
+
+const idType = z.coerce.bigint().positive()
+const BoundarySchema = z.object({
+  ids: z.union([z.array(idType), idType.transform((x) => [x])]),
```

Comment:
@elsueno I had issues with the `ids` parameter, as it was not type consistent. So this was my way around. If you know a better solution lmk.

Reply:
The problem was that providing a single id resulted in `ids:String` and providing more than one id resulted in `ids:String[]`